test/elpa.el

```diff
@@ -0,0 +1,5 @@
+(setq package-user-dir
+      (expand-file-name (format ".cask/%s/elpa" emacs-version)))
```

Comment:
Finally I'd prefer if you did not use .cask to install packages manually.  We make no guarantees about the layout of that folder, and I'd not like to see users being confused by a name that refers to a tool that's not being used.

If anything breaks in your code it given the name it might end up in our tracker and generate additional work for us.